packages/thriver-accounts/lib/templates/profile.html

```diff
@@ -1,6 +1,202 @@
 <template name="profile">
-  <header><h2>{{heading}}</h2></header>
-  {{> quickForm collection="Meteor.users" id="updateProfile" doc=doc type="update"
-    omitFields=omitFields fields="profile" validation="browser" 
-      buttonContent=submitValue}}
-</template>
+  <form id="profile">
+    <header>
+      <h2>{{heading}}</h2>
+    </header>
+    <div class="form-group-container">
+      {{ #each user.emails}}
```

Comment:
There seems to be a lot of inconsistencies surrounding bracket styles.  I think everywhere else in the codebase (though we have no linting for markup) we leave out spaces:  `{{something}}` except for templates:  `{{> template}}`.